provisioner.go

```diff
@@ -73,14 +73,14 @@ func (p *defaultProvisioner) Provision(ctx context.Context, appName, os, arch st
 		err = errors.Wrap(err, "provisioning astilectron failed")
 		return
 	}
-	s.Astilectron = &ProvisionStatusPackage{Version: VersionAstilectron}
+	s.Astilectron = &ProvisionStatusPackage{Version: paths.VersionAstilectron}
```

Comment:
I don't really like the idea of storing the versions in the `Paths` struct.

I think the better solution would be to update the `Provisioner` interface into `Provision(ctx context.Context, appName, os, arch, versionAstilectron, versionElectron string, p Paths) error`.

You would need to update more stuff.

Reply:
Removed storing of versions in Paths. Updated function signatures, to pass versions